$OpenBSD: patch-gtk_window-decorator_gtk-window-decorator_c,v 1.2 2010/09/30 07:23:47 ajacoutot Exp $

XXX make it compile with gtk+2 2.22 (*_DISABLE_DEPRECATED)
gtk-window-decorator.c:5626: error: 'gdk_display' undeclared (first use in this function)

--- gtk/window-decorator/gtk-window-decorator.c.orig	Sun Mar  1 13:07:14 2009
+++ gtk/window-decorator/gtk-window-decorator.c	Wed Sep 29 21:11:16 2010
@@ -31,10 +31,6 @@
 #include <X11/extensions/Xrender.h>
 #include <X11/Xregion.h>
 
-#ifndef GDK_DISABLE_DEPRECATED
-#define GDK_DISABLE_DEPRECATED
-#endif
-
 #ifndef GTK_DISABLE_DEPRECATED
 #define GTK_DISABLE_DEPRECATED
 #endif
@@ -4233,7 +4229,7 @@ show_tooltip (const char *text)
 static void
 hide_tooltip (void)
 {
-    if (GTK_WIDGET_VISIBLE (tip_window))
+    if (gtk_widget_get_visible (tip_window))
 	g_get_current_time (&tooltip_last_popdown);
 
     gtk_widget_hide (tip_window);
